scalding-serialization/src/main/scala/com/twitter/scalding/serialization/OrderedSerialization.scala

```diff
@@ -101,6 +101,30 @@ object OrderedSerialization {
     case NonFatal(e) => CompareFailure(e)
   }
 
+  def viaTransform[T, U](
+    packFn: T => U,
+    unpackFn: U => T)(implicit otherOrdSer: OrderedSerialization[U]): OrderedSerialization[T] =
+    new OrderedSerialization[T] {
```

Comment:
Should this be a new type like `TransformedOrderedSerialization`? Might be useful for optimizing serialization chains, or maybe this is not applicable.